src/front/app/components/users/SampleListUsers.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file Sample list users component.
+ * This file has no worry for styles and is only here to serve the purpose of showing
+ * how to implement Apollo. Check the file
+ * /containers/sampleListUsers to see where most props are coming from.
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class SampleListUsers extends Component {
+  static propTypes = {
+    users: PropTypes.arrayOf(PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      name: PropTypes.string.isRequired,
+      username: PropTypes.string.isRequired,
```

Comment:
based on the files below, the `email` property is also expected within the shape of users data.

Reply:
Not really using it in the component, so no need to do so, but I'll add it if it makes you happy.